apps/service-portal/src/components/UserMenu/UserMenu.tsx

```diff
@@ -45,57 +51,60 @@ const UserMenu = ({ state, onClose, onRouteChange }: UserMenuProps) => {
       setDropdownState={onClose}
       onLogout={signOut}
       dropdownItems={
-        <Box>
-          <Box
-            display="flex"
-            alignItems="center"
-            marginBottom={1}
-            className={styles.breakWord}
-          >
-            <Box display="flex" alignItems="center" marginRight={2}>
-              <Icon type="outline" icon="mail" color="blue300" />
+        <>
+          <Box>
+            <Box
+              display="flex"
+              alignItems="center"
+              marginBottom={1}
+              className={styles.breakWord}
+            >
+              <Box display="flex" alignItems="center" marginRight={2}>
+                <Icon type="outline" icon="mail" color="blue300" />
+              </Box>
```

Comment:
FYI: the `<Inline />` component is good for this.

Reply:
Just indenting. 👍